Header header
uint8[] digits
